packages/lesswrong/components/hooks/useDialogNavigation.ts

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useRef } from 'react';
+
+interface UseDialogNavigationOptions {
+  /**
+   * Whether to track if the dialog is closing via the back button.
+   * When true, returns an isClosingViaBackRef that can be used to determine
+   * if the dialog is closing via browser navigation.
+   */
+  trackClosingViaBack?: boolean;
```

Comment:
Isn't `trackClosingViaBack: true` the behavior at both call-sites?  Also, neither callsite uses the returned ref.  I think for now you should just get rid of the configuration.